FirebaseRTC/public/prWaitingRoom.html

```diff
@@ -22,10 +22,10 @@
 
      <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-     <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+     <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/font-awesome.css">
+    <link rel="stylesheet" type="text/css" href="/assets/csslatemo-art-factory.css">
```

Comment:
Same issue with "/temp" being lost in the file pathway change. See comment in patWaitingRoom.html file

Reply:
The issue is fixed in the new commit.